ansible/playbooks/roles/filebeat/templates/custom-chart-values.yml.j2

```diff
@@ -50,39 +50,19 @@ filebeatConfig:
 {# -------------------------- Kubernetes input -------------------------- #}
 
 {% if (k8s_as_cloud_service is defined and k8s_as_cloud_service) or ('kubernetes_master' in groups or 'kubernetes_node' in groups) %}
-    - type: docker
+    - type: container
       enabled: true
-      containers.ids: "*"
-{% if specification.docker_input.multiline is defined %}
+{% if specification.container_input.multiline is defined %}
       multiline:
-{% for k, v in specification.docker_input.multiline.items() %}
+{% for k, v in specification.container_input.multiline.items() %}
         {{ k }}: {{ v }}
 {% endfor %}
 {% endif %}
 
       processors:
-        - add_docker_metadata:
-        - rename:
-            fields:
-              - from: docker.container.labels.io.kubernetes.container.name
-                to: kubernetes.container.name
-              - from: container.labels.io_kubernetes_container_name
-                to: kubernetes.container.name
-
-              - from: docker.container.labels.io.kubernetes.pod.name
-                to: kubernetes.pod.name
-              - from: container.labels.io_kubernetes_pod_name
-                to: kubernetes.pod.name
-
-              - from: docker.container.labels.io.kubernetes.pod.namespace
-                to: kubernetes.namespace
-              - from: container.labels.io_kubernetes_pod_namespace
-                to: kubernetes.namespace
-            ignore_missing: true
-            fail_on_error: true
-        - drop_fields:
-            fields:
-              - docker
+        - add_kubernetes_metadata:
```

Comment:
Does it provide the same fields as we had so far (`kubernetes.container.name`, `kubernetes.pod.name`, `kubernetes.namespace`)? If names are different, it would be nice to inform about this in changelog (breaking changes for Kibana dashboards).